src/machines/java/com/enderio/machines/common/init/MachineBlockEntities.java

```diff
@@ -55,14 +56,13 @@ public class MachineBlockEntities {
         register("fluid_tank", FluidTankBlockEntity.Standard::new, MachineBlocks.FLUID_TANK)
             .setRenderer(() -> FluidTankBER::new)
             .apply(MachineBlockEntities::machineBlockEntityCapabilities)
-            .addCapability(Capabilities.FluidHandler.BLOCK, FluidTankUser.FLUID_HANDLER_PROVIDER);
-
```

Comment:
I personally don't think this is needed, but it doesn't hurt.

Reply:
I forgot to add the fluid cap to again